Libraries/Image/ImageStylePropTypes.js

```diff
@@ -55,6 +55,12 @@ var ImageStylePropTypes = {
    * @platform android
   */
   overlayColor: ReactPropTypes.string,
+
+  // Android-Specific styles
+  borderTopLeftRadius: ReactPropTypes.number,
+  borderTopRightRadius: ReactPropTypes.number,
+  borderBottomLeftRadius: ReactPropTypes.number,
+  borderBottomRightRadius: ReactPropTypes.number,
```

Comment:
Sorry @kmagiera, it seems that I was mistaken here: these properties are not currently supported directly on the `Image` component on iOS, but you can create this same effect by giving a specific corner radius to a wrapper View and setting overflow to hidden.

```
<View style={styles.test}>
  <Image
    style={styles.image}
    source={{uri: 'https://www.petfinder.com/wp-content/uploads/2012/11/140272627-grooming-needs-senior-cat-632x475.jpg'}}
  />
</View>
```

```
test: {
  borderTopLeftRadius: 20,
  overflow: 'hidden',
},
```

Results in:

<img width="777" alt="screen shot 2016-03-25 at 12 18 03 pm" src="https://cloud.githubusercontent.com/assets/90494/14051941/a5dd0178-f283-11e5-87fd-4224857a9a39.png">

Assigning the border property specifically to an `img` tag on the web will work as suggested by this PR:

<img width="848" alt="screen shot 2016-03-25 at 12 18 31 pm" src="https://cloud.githubusercontent.com/assets/90494/14051956/c2c5b5c8-f283-11e5-9c05-cfbb317afe55.png">